quarkus/tests/integration/src/test/resources/org/keycloak/it/cli/approvals/cli/help/HelpCommandTest.testBuildHelp.unix.approved.txt

```diff
@@ -30,6 +30,7 @@ Cluster:
                        Default: ispn.
 --cache-config-file <file>
                      Defines the file from which cache configuration should be loaded from.
+                       The configuration file is relative to the 'conf/' directory.
```

Comment:
The runtime will apply word-wrapping, therefore this test fails. 

The "The" will be at the previous line; and it will have a blank if it has been added to the Java code (as described above). Applies to all files, including the windows files. This is a manual edit, please run the tests locally (if possible), or ping me if I should run them.

```suggestion
                     Defines the file from which cache configuration should be loaded from. The
                       configuration file is relative to the 'conf/' directory.
```